Week-1/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Page Title</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./style.css"> 
+    
+  </head>
+  <body>
+    <div class="wraper">
```

Comment:
@SmitSheth01 no use of web semantics. Please make use of header, sections, nav, sidebars etc